varsim.py

```diff
@@ -119,7 +119,9 @@ def monitor_multiprocesses(processes, logger):
             is_success = False
         else:
             logger.info("Process with pid %d finished successfully" % p.pid)
-    return is_success
+    if not is_success:
+        raise Exception('One process aborted. Please check log for details.');
```

Comment:
Yeah, that would be nice! I did want to spruce up the multithreading capabilities.